src/lib/ViewerState.ts

```diff
@@ -24,7 +24,7 @@ export class ViewerState {
     showTracks = true;
     showTrackHighlights = true;
     // Default position and target for the camera
-    cameraPosition = [0, 0, -4]; // was  [500, 500, -1250];
+    cameraPosition = [-4, 0, 0]; // was  [-1250, 500, 500];
```

Comment:
Why did these get flipped in this PR?

Reply:
The coordinates in the `points` Zarr array are saved as [z,y,x,...,z,y,x], but when loaded and rendered, the opposite convention is used (`positions.setXYZ(i, data[num * i + 0], data[num * i + 1], data[num * i + 2]);` in `pointCanvas.ts/setPointsPositions`). This means that the XYZ axes are actually ZYX. This only means that the point cloud is effectively rotated, which is never a problem because absolute coordinates were not relevant before. When fixing the camera to XY in the case of a 2D dataset, this had to be changed. A better solution would be to change the plotting of the coordinates, but this involves a bit more work, since this also holds for the tracks etc, I will look into this in another PR.